pkg/vpcmodel/explainabilityPrint.go

```diff
@@ -206,6 +213,12 @@ func (g *groupedConnLine) explainPerCaseStr(c *VPCConfig, src, dst EndpointElem,
 	case crossVpcRouterRequired(src, dst) && crossVpcRouter != nil && crossVpcConnection.IsEmpty():
 		return fmt.Sprintf("%vAll connections will be blocked since transit gateway denies route from source to destination"+tripleNLVars,
 			noConnection, headerPlusPath, details)
+	case isAtPrivateSubnet(dst) && src.IsExternal():
```

Comment:
The cases in which we provide a single line explaining why there is no connection should be minimized see #655. In this specific case we can provide full explainability and integrate the private subnet in the path and details. 

We could defer the explanation part of this PR to latter PRs - I can take responsibility to this part.

Reply:
OK, will be resolved in another PR, by our explainability expert.
added a bullet to the relevant issue https://github.com/np-guard/vpc-network-config-analyzer/issues/756

Reply:
Perhaps better remove it all together